tests/filtering/test_taxonomy_filtering.py

```diff
@@ -0,0 +1,107 @@
+from unittest import TestCase
+
+import pandas as pd
+import numpy as np
+
+from moonstone.filtering.taxonomy_filtering import TaxonomyNamesFiltering
+
+
+class TestTaxonomyNamesFilteringg(TestCase):
```

Comment:
You have a typo here. Don't think you'll use this class anywhere else, so it won't really matter but just in case